co-simulation/bundle/src/assembly/resources/scenarios/Town04/sumo/bus_stop.add.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- generated on 2023-11-06 15:52:06 by Eclipse SUMO netedit Version 1.18.0
+-->
+
+<additional xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/additional_file.xsd">
+    <!-- StoppingPlaces -->
+    <busStop id="bs_2" lane="10_3" startPos="2.00" endPos="8.00"/>
+</additional>
```

Comment:
If we are not using bus stop, do we still need this? or is also being used for the heavy_vehicles to stop?

Reply:
Yes, it's actually used in the rou.xml file, under the third car